src/pages/index.js

```diff
@@ -45,121 +43,157 @@ const SEO_TITLE = 'The easiest way to get Spotify Backstage';
 const LEAD = `Roadie's SaaS platform handles hosting and upgrades and ensures you always have access to the latest Backstage features.`;
 
 const PRODUCT = {
-  features: [{
-    title: 'Quick and easy setup',
-    description: 'Add plugins in a couple of clicks. Drag widgets to the place you want them.',
-    illustration: {
-      webp: DragDropIllustration,
-      png: DragDropIllustrationPng,
-      alt: 'A component from a Backstage plugin being dragged around with the mouse pointer',
-    },
-    bullets: [{
-      name: 'Custom plugin support',
-      description: 'Building your own internal plugins? Publish them to our private repository and they appear where you need them.',
-      icon: BeakerIcon,
-    }, {
-      name: 'Secure connections',
-      description: 'Securely connect to SaaS services and your own infrastructure using our tunneling broker service.',
-      icon: ShieldCheckIcon,
-    }],
-
-  }, {
-    title: 'Maintenance free',
-    description: 'Roadie handles upgrades and security so you can stay focused.',
-    illustration: {
-      webp: SecurityMaintenanceIllustration,
-      png: SecurityMaintenanceIllustrationPng,
-      alt: 'A progress bar and cog to indicate an upgrade in progress.',
-    },
-    bullets: [{
-      name: 'Automated upgrades',
+  features: [
+    {
+      title: 'Focus on building value instead of learning Backstage internals',
       description:
-        `Open-source means Backstage moves quickly. Upgrades can take days, and cause downtime when they go wrong. Why spend the time?`,
-      icon: ChipIcon,
-    }, {
-      name: 'Security patches delivered',
+        'Roadie manages deployments and security for you and helps you get the best out of Backstage with a few clicks.',
+      illustration: {
+        webp: DragDropIllustration,
+        png: DragDropIllustrationPng,
+        alt: 'A component from a Backstage plugin being dragged around with the mouse pointer',
+      },
+      bullets: [
+        {
+          name: 'Secure connections',
+          description:
+            'Securely connect to SaaS services and your own infrastructure using our tunneling broker service.',
+          icon: ShieldCheckIcon,
+        },
+        {
+          name: 'Custom plugin support',
+          description:
+            'Building your own internal plugins? Publish them to our private repository and they appear where you need them.',
+          icon: BeakerIcon,
+        },
+      ],
+    },
+    {
+      title: 'Keep your Backstage ever-green',
       description:
-        `We regularly patch vulnerabilities in the open-source code, ensuring you're using a hardened version of Backstage.`,
-      icon: ShieldCheckIcon,
-    }],
-  }],
+        'All Roadie customers are running on the latest Backstage version (1.3.0). Get the latest features from the community without the maintenance burden.',
```

Comment:
This version is already out of date. I think it will be quite burdensome to have to remember to update this all the time.